src/instruments/src/Common/gauges.tsx

```diff
@@ -171,3 +173,124 @@ export const VerticalSegment: FC<VerticalSegmentProps> = ({ x, y, height, rangeS
+    return (decimalSplit);
+};
+
+export const valueRadianAngleConverter = (value, min, max, endAngle, startAngle) => {
+    const valuePercentage = (value - min) / (max - min);
+    // const start = startAngle > 180 ? 360 - startAngle : startAngle;
+    let angleInRadians = startAngle > endAngle
+        ? startAngle + (valuePercentage * (360 - startAngle + endAngle)) - 90
+        : startAngle + (valuePercentage * (endAngle - startAngle)) - 90;
+    // let angleInRadians = angleInRadiansOne > 359 ? 360 - angleInRadiansOne : angleInRadiansOne;
+    console.log(`Angle is ${angleInRadians}`);
+    angleInRadians *= (Math.PI / 180.0);
+    console.log(`Angle is radians is ${angleInRadians}`);
+    return ({
+        x: Math.cos(angleInRadians),
+        y: Math.sin(angleInRadians),
+    });
+};
+
+type GaugeMarkerComponentType = {
+    value: number,
+    x: number,
+    y: number,
+    min: number,
+    max: number,
+    radius: number,
+    startAngle: number,
+    endAngle: number,
+    className: string,
+    showValue?: boolean,
+    indicator?: boolean,
+    outer?: boolean,
+    multiplier?: number,
+};
+
+const GaugeMarkerComponentNoMemo = ({ value, x, y, min, max, radius, startAngle, endAngle, className, showValue, indicator, outer, multiplier } : GaugeMarkerComponentType) => {
+    const dir = valueRadianAngleConverter(value, min, max, endAngle, startAngle);
+
+    if (typeof multiplier === 'undefined') multiplier = 1.15;
+
+    let start = {
+        x: x + (dir.x * radius * 0.9),
+        y: y + (dir.y * radius * 0.9),
+    };
+    let end = {
+        x: x + (dir.x * radius),
+        y: y + (dir.y * radius),
+    };
+
+    if (outer) {
+        start = {
+            x: x + (dir.x * radius),
+            y: y + (dir.y * radius),
+        };
+        end = {
+            x: x + (dir.x * radius * multiplier),
+            y: y + (dir.y * radius * multiplier),
+        };
+    }
+
+    if (indicator) {
+        start = { x, y };
+
+        end = {
+            x: x + (dir.x * radius * multiplier),
+            y: y + (dir.y * radius * multiplier),
+        };
+    }
+
+    // Text
+    const multiplierX = value >= 8 ? 0.8 : 0.7;
+    const multiplierY = value >= 8 ? 0.52 : 0.7;
+    const pos = {
+        x: x + (dir.x * (radius * multiplierX)),
+        y: y + (dir.y * (radius * multiplierY)),
+    };
+
+    const textValue = !showValue ? '' : Math.abs(value).toString();
+
+    return (
+        <>
+            <line x1={start.x} y1={start.y} x2={end.x} y2={end.y} className={className} />
+            <text x={pos.x} y={pos.y} className={className} alignmentBaseline="central" textAnchor="middle">{textValue}</text>
+        </>
+    );
+};
+export const GaugeMarkerComponent = React.memo(GaugeMarkerComponentNoMemo);
+
+interface GaugeComponentProps {
+    x: number,
+    y: number,
+    radius: number,
+    startAngle: number,
+    endAngle: number,
+    className: string,
+    manMode: boolean,
+    children?: React.ReactNode,
```

Comment:
**suggestion**: remove `children` from the type

```suggestion 
```

Reply:
GaugeMarkers are children of a Gauge component, so this needs to stay.

Reply:
Gauge Markers are a child of Gauge, so this is required.

Reply:
No, you already use FC which gives your component the ability to contain children.

Reply:
In addition, speaking of GaugeComponent, can you please rename these components to not have the word ‘Component’ in their name?

Reply:
I'd rather we change that in a separate PR if need be

Reply:
> I'd rather we change that in a separate PR if need be

How come?

Reply:
Hmmm, if I remove reference to children I get this error:
Type '{ children: Element[]; x: number; y: number; radius: number; startAngle: number; endAngle: number; manMode: true; className: string; }' is not assignable to type 'IntrinsicAttributes & GaugeComponentProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & GaugeComponentProps'.ts(2322)
(alias) const GaugeComponent: React.NamedExoticComponent<GaugeComponentProps>
import GaugeComponent

Reply:
Nevermind, sorted it.

Reply:
Awesome! Other than that, just remove any existing console.logs after any last-minute testing you need to do if-any and that should be good :)

Reply:
> 
> 
> > I'd rather we change that in a separate PR if need be
> 
> How come?

Wanna keep this PR focused.